tests/Behat/MediaContext.php

```diff
@@ -82,6 +108,176 @@ public function visitMediaPage(string $name): void {
     $this->visitPath($media->toUrl()->toString());
   }
 
+  /**
+   * Creates media documents with the specified file names.
+   *
+   * Usage example:
+   *
+   * Given the following documents:
+   *   | name   | file   |
+   *   | name 1 | file 1 |
+   *   |   ...  |   ...  |
+   *
+   * @Given the following documents:
+   */
+  public function createMediaDocuments(TableNode $file_table): void {
+    // Retrieve the url table from the test scenario.
+    $files = $file_table->getColumnsHash();
+    foreach ($files as $properties) {
+      $file = $this->createFileEntity($properties['file']);
+      $media = \Drupal::service('entity_type.manager')
+        ->getStorage('media')->create([
+          'bundle' => 'document',
+          'name' => $properties['name'],
+          'oe_media_file' => [
+            'target_id' => (int) $file->id(),
+          ],
+          'status' => 1,
+        ]);
+      $media->save();
+
+      // Store for cleanup.
+      $this->medias[] = $media;
+    }
+  }
+
+  /**
+   * Creates media documents with the specified file names.
+   *
+   * Usage example:
+   *
+   * Given the following images:
+   *   | name   | file   | alt   | title   |
+   *   | name 1 | file 1 | alt 1 | title 1 |
+   *   | ...    | ...    | ...   | ...     |
+   *
+   * Properties "alt" and "title" are optional, "name" will be used if they
+   * are not provided.
+   *
+   * @Given the following images:
+   */
+  public function createMediaImages(TableNode $file_table): void {
+    // Retrieve the url table from the test scenario.
+    $files = $file_table->getColumnsHash();
+    foreach ($files as $properties) {
+      $file = $this->createFileEntity($properties['file']);
+      $media = \Drupal::service('entity_type.manager')
+        ->getStorage('media')->create([
+          'bundle' => 'image',
+          'name' => $properties['name'],
+          'oe_media_image' => [
+            'target_id' => (int) $file->id(),
+            'alt' => $properties['alt'] ?? $properties['name'],
+            'title' => $properties['title'] ?? $properties['name'],
+          ],
+          'status' => 1,
+        ]);
+      $media->save();
+
+      // Store for cleanup.
+      $this->medias[] = $media;
+    }
+  }
+
+  /**
+   * Creates media AVPortal photos with the specified URLs.
+   *
+   * Usage example:
+   *
+   * Given the following AV Portal photos:
+   *   | url   |
+   *   | url 1 |
+   *   |  ...  |
+   *
+   * @Given the following AV Portal photos:
```

Comment:
```suggestion
   * @Given the following AV Portal photo(s):
```
would be nice to have